docs/concepts/staking/block_reward.md

```diff
@@ -1,21 +1,70 @@
 ---
 tags:
+  - block reward
   - Staking
 ---
 
 
 # Staking
 
 
-## Block reward
+## Block Reward
+---
+
+Block rewards in a blockchain serve as an incentive for validators who actively contribute to the creation of new blocks within the blockchain network. This rewarding process, commonly referred to as validation, is particularly relevant in **delegated proof-of-stake (DPoS)** blockchains.
+
+In DPoS blockchains, validators are selected to validate transactions and generate new blocks based on the quantity of XPLL they have staked within the network. As a reward for their contributions, validators receive a share of the transaction fees and a portion of the block rewards for successfully creating new blocks. 
+
+Block rewards play a crucial role in blockchain networks by serving multiple purposes. They incentivize miners and validators' participation in the network's consensus process, thereby upholding the security and integrity of the network. Additionally, block rewards facilitate the introduction of new XPLL into circulation, ensuring the long-term sustainability of the network.
+
+
+Reward formulas can be found in the technical documentation [parallelchain-protocol](https://github.com/parallelchain-io/parallelchain-protocol/blob/master/Runtime.md).
+
+### Reward Rate
+
+The block reward amount gradually decreases as additional blocks are incorporated into the blockchain over time. This process, which is known as the **block reward halving**, is a characteristic found in various blockchain networks, such as Bitcoin. By implementing the block reward halving, the supply of cryptocurrency is regulated, effectively curbing inflationary pressures.
+
+The reward rate is calcuated as following:
+
+- First year: 8.5% per annum
+- Reduce 15% per annum
+- After 10 year (3650 days): keep at 1.5% per annum
```

Comment:
Nitpick: 10 years - which can actually be more than 3650 days

Reply:
Additionally, the formula for the issuance rate results in only 8% inflation per annum. I calculated this with compounding interest on each epoch with following python code:

`totaltokens = 1
for n in range(1, 366):
    totaltokens += totaltokens * (0.0835 * 0.85**(n/365) / 365)

print(((totaltokens / 1) -1 ) * 100)
// yields 8.008813021222627
`

Can you verify that either the inflation rate is only 8% p.a in the first year, or that the initial factor is higher  (should be around 0.0885 for 8.5% p.a. in the first year.

Thanks

Reply:
I re-checked and the issuance rate is indeed supposed to be 8% per annum in the first year.

The docs and the protocol spec must have mixed up the 8% target issuance rate with the 0.85 term used to reduce the issuance rate by 15% each year.

Thank you for pointing this out. We’ll update the docs and protocol spec promptly.